lib/rules/no-this-render.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const emberUtils = require('../utils/ember');
+const types = require('../utils/types');
+
+const ERROR_MESSAGE = "Do not use `this.render(). Prefer @ember/test-helpers' `render` instead.`";
```

Comment:
Can you change this wording slightly for clarity:

Before:
```
Prefer @ember/test-helpers' `render` instead.
```

After:
```
Prefer `render` from @ember/test-helpers instead.
```

Reply:
Done, I also added backticks around `@ember/test-helpers`.